test/images/scripts/virt-builder-fedora

```diff
@@ -32,6 +32,8 @@ if [ "$#" -gt 3 ]; then
     subscription_args="--mkdir /root/.rhel \
       --upload $subscription_path/login:/root/.rhel/login \
       --upload $subscription_path/pass:/root/.rhel/pass"
+else
+    subscription_args="--touch /root/.skip_repos"
```

Comment:
Hi,
yes, exactly used by me, for creating own images with own internal repositories.